connectors/sources/postgresql.py

```diff
@@ -11,8 +11,6 @@
 
 from connectors.sources.generic_database import GenericBaseDataSource
 
-# Below schemas are system schemas and the tables of the systems schema's will not get indexed
-SYSTEM_SCHEMA = ["pg_toast", "pg_catalog", "information_schema"]
```

Comment:
Probably good to disallow these schemas from extraction still, what do you think?

Reply:
Maybe a customer wants to ingest those for reasons we (or at least I) don't know. Therefore I've just removed this assumption that it's _never_ ingested from the code as this needs to be explicitly configured anyway. I think we can easily put it in again, if there's a need. WDYT?

Reply:
Yeah I don't have a really strong stance. I can see export of metrics from `information_schema` can be useful, but not in our use-case - they could work well if we appended them instead of overwriting data in the index to have time-series data over performance.

So I doubt anyone will sync those, but if we don't validate this it should not be a problem.

Reply:
It's good defaults though -- I don't think anyone would ever want to index these, can't think of a use case, it's internal PG data. Maybe this connector could have an option with a list of tables to ignore, with these 3 by default.